src/components/Tag.tsx

```diff
@@ -2,7 +2,7 @@ import { Icon, IconKey } from "src/components";
 import { Css, Margin, Only, Xss } from "src/Css";
 import { useTestIds } from "src/utils";
 
-type TagXss = Margin;
```

Comment:
**question:** Where did the need for custom background colors come from regarding our tags? Do we have a ticket or Figma regarding this?
The way Tags were intended to work is that we had a subset of supported background colors that provided a specific meaning. By allowing any background color then we are enabling inconsistency across the application.
If we want to allow a new background color, then I would suggest we add a new `TagType`.

Reply:
Ididn't need it, I didn't know that the tagType change the color, I need it the info one, I'll remove it on my next PR

Reply:
Awesome. Thanks for following up!